benchmarks/file_benchmark.py

```diff
@@ -40,14 +40,26 @@ def run(self, source, case=None, **kwargs):
                     yield self.r_benchmark(command, tags, kwargs, case)
 
     def _get_r_command(self, source, case, options):
+        # changed param names to align with Python in version 0.1.1
+        is_legacy_str, _ = self.conbench.execute_r_command(
+            'cat(packageVersion("arrowbench") <= "0.1.0")'
+        )
+        is_legacy = is_legacy_str == "TRUE"
```

Comment:
New arrowbench version will bump version to 0.1.1 and change these param names to align with the Python so tag names will not change in Conbench